apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/parser/graphql/ast/document.kt

```diff
@@ -0,0 +1,128 @@
+package com.apollographql.apollo.compiler.parser.graphql.ast
```

Comment:
nit: is there a way to better name this file, like `documentParser`?

Reply:
I renamed to `GraphQLParser` in #2756